gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -148,7 +148,8 @@
    */
   public enum OutputStreamType {
     BASIC,
-    SYNCABLE_COMPOSITE
+    SYNCABLE_COMPOSITE,
+    FLUSHABLE_COMPOSITE
```

Comment:
Put `FLUSHABLE_COMPOSITE` on 2nd place to alphabetically sort values

Reply:
done